README.md

```diff
@@ -60,77 +61,66 @@ export ARTIFACTORY_USERNAME=firstname.lastname
 export ARTIFACTORY_PASSWORD=Long_string_of_gibberish_abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUV
 ```
 
-- `ARTIFACTORY_USERNAME`: 
+- `ARTIFACTORY_USERNAME`:
   Find your username in [JFrog](https://digitalasset.jfrog.io/ui/admin/artifactory/user_profile); usually `firstname.lastname`.
-- `ARTIFACTORY_PASSWORD`: 
+- `ARTIFACTORY_PASSWORD`:
   This is your **API Key**, *NOT* your password. Find your API key at the bottom of **Authentication Settings** in your profile.
 
-## Build and view the docs locally
-
-`cd` to the `docs` directory in the repository:
-
-```zsh
-cd docs
-```
-
 ### Download docs
 
 Run the following script to download the documentation tarballs from the Daml and Canton repos:
 
 ```zsh
-./download.sh
+download
```

Comment:
wasn't able to run these but assuming the change is in this PR...so will test later ...

Reply:
Yes, the change is in this PR.